ee/clickhouse/views/test/test_clickhouse_experiments.py

```diff
@@ -639,6 +640,7 @@ def test_creating_updating_experiment_with_group_aggregation(self):
         self.assertEqual(created_ff.filters["aggregation_group_type_index"], None)
 
 
+@flaky(max_runs=3, min_passes=1)
```

Comment:
I recommend max_runs=10

Reply:
Any specific reason for this?

The flake is generally rare (1 in 50ish runs, I've never seen it flake on my PRs in the past week), and if the probability is > 33%, I'd like to know about it! (vs > 10% when max_runs = 10)

Reply:
Cost of flakes on CI is huge - it _will_ slow down peoples development workflows significantly as they need to stop the world and go and dig into _why_ things are failing.

The less of this the better - there's 0 win to be had if we have more frequent fails. No comments on local development though.